source/edge-server/connect.txt

```diff
@@ -201,27 +227,82 @@ You can connect to the Edge Server using standard MongoDB Drivers and tools.
 Clients use a specialized MongoDB URI connection string to connect to 
 Edge Server and send requests.
 
-The wire protocol server is on by default. You can connect to it via the 
-host machine using this MongoDB URI:
+The Wire Protocol server accepts these incoming connections, and is on by 
+default. You can connect to it via the host machine using this MongoDB URI:
 
 .. code:: console
 
-   mongodb://EDGE_SERVER_HOSTNAME:27021
+   mongodb://<username>:<password>@<hostname>:<port>?authMechanism=PLAIN&appName=::<auth-type>
+
+.. list-table::
+   :header-rows: 1
+
+   * - Element
+     - Description
+
+   * - ``<username>``
+     - The username of the Atlas user. For an 
+       :ref:`email/password user <email-password-authentication>`, this 
+       might look similar to ``first.last@example.com``.
+
+   * - ``<password>``
+     - The password of the Atlas user. This might look similar to 
+       ``myPassword123456``.
```

Comment:
I added it to continue the pattern of providing examples, and because anecdotally some docs users report getting confused about whether to leave in things like `<>` when they see example strings. So when I use special characters to provide context, I try to always provide an example showing the proper formatting of the value that omits the special characters.